cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -104,7 +104,7 @@ func livenessProbe(host string, port int, scheme v1.URIScheme) *v1.Probe {
 }
 
 // CreateStaticPodFiles creates all the requested static pod files.
-func CreateStaticPodFiles(manifestDir string, cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint, componentNames ...string) error {
+func CreateStaticPodFiles(kubernetesDir, manifestDir string, cfg *kubeadmapi.ClusterConfiguration, endpoint *kubeadmapi.APIEndpoint, componentNames ...string) error {
```

Comment:
It's odd for this to take both `manifestDir` and `kubernetesDir`. Can we avoid that?